pramen-py/src/pramen_py/utils/file_system.py

```diff
@@ -52,10 +55,25 @@ def get_fs_from_uri(self, uri: str) -> "_FileSystem":
         object (i.e. S3FileSystem, LocalFileSystem etc.)
         """
         return self.FileSystem.get(
-            self.URI(uri),
+            self.URI(self.fix_uri_scheme_availability(uri)),
```

Comment:
We are expecting the properly formatted uri at this place. The postprocessing of the config input is happening here https://github.com/AbsaOSS/pramen/blob/main/pramen-py/src/pramen_py/models/__init__.py#L73-L83.